incubator/scylladb/cluster-admin.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: cluster-admin
+  annotations:
+    rbac.authorization.kubernetes.io/autoupdate: "true"
+rules:
```

Comment:
This is not a very good practice.
Please configure fine grained access to resources as needed

Reply:
To be honest with you being new to k8s/helm rbac was a little overwhelming so I just created the basics to make it work. I was hoping to dig into rbac and making it more safe in a next version. But if you tell me this is an absolute must for the chart to get accepted I will gladly do it. (Might take me a few days).